crates/containerd-shim-wasmtime-v1/src/sandbox/oci.rs

```diff
@@ -28,6 +30,17 @@ pub fn load(path: &str) -> Result<Spec, Error> {
     Ok(spec)
 }
 
+pub fn get_root(spec: &Spec) -> Result<&PathBuf, Error> {
+    let root = spec.root().as_ref().unwrap();
```

Comment:
Doesn't seem like we are using the error return in this function.

Reply:
fixed